src/components/SelectComponents/Select/Select.tsx

```diff
@@ -16,97 +14,103 @@ interface Props<S> extends Omit<React.HTMLAttributes<HTMLDivElement>, 'onChange'
-        onBlur?.();
-    };
+    const selectionRenderer = () => (
+        <span {...elem('selected')}>
+            {selectedItem ? (
+                itemToString(selectedItem)
+            ) : (
+                <Text inline {...elem('placeholder')} context="muted">
+                    {placeholder}
+                </Text>
+            )}
+        </span>
+    );
 
-    const { isOpen, getToggleButtonProps, getMenuProps, highlightedIndex, getItemProps } =
-        useSelect({
-            items,
-            selectedItem,
-            onSelectedItemChange: handleSelection,
-        });
+    const blurredRenderer = () => (
+        <div tabIndex={0} role="searchbox" onFocus={onFocus} {...elem('wrapper')}>
+            {selectionRenderer()}
+        </div>
+    );
 
-    const handleToggle = (e) => {
-        e?.stopPropagation();
-        if (isOpen) {
-            onBlur?.();
-        } else {
-            onFocus?.();
-        }
-    };
+    const focusedRenderer = ({ getToggleButtonProps, onBlur: blur }) => (
+        <div
+            tabIndex={0}
+            role="searchbox"
+            {...elem('wrapper', { isOpen: true })}
+            {...getToggleButtonProps({ onClick: blur })}
+        >
+            {selectionRenderer()}
+        </div>
+    );
 
     return (
-        <div ref={rootRef} {...rest} {...block({ className })}>
-            <div {...elem('main', { isOpen })}>
-                <FieldWrapper isFocused={isOpen} {...elem('field')}>
-                    <div
-                        tabIndex={0}
-                        role="searchbox"
-                        {...elem('wrapper', { isOpen })}
-                        {...getToggleButtonProps({ onClick: handleToggle })}
-                    >
-                        <span {...elem('selected')}>{itemToString(selectedItem)}</span>
-                        {isOpen ? (
-                            <IoMdArrowDropup {...elem('dropdownIcon')} />
-                        ) : (
-                            <IoMdArrowDropdown {...elem('dropdownIcon')} />
-                        )}
-                    </div>
-                    <List
-                        {...getMenuProps({ ref: listRef })}
-                        {...elem('list', { isOpen })}
-                        isControlledNavigation
-                    >
-                        {isOpen && (
-                            <SuggestionsList
-                                suggestionToString={itemToString}
-                                suggestionItemRenderer={optionItemRenderer}
-                                suggestions={items}
-                                getItemProps={getItemProps}
-                                highlightedIndex={highlightedIndex}
-                                passDisabledToListItems
-                            />
-                        )}
-                    </List>
-                </FieldWrapper>
-            </div>
+        <div {...rest} {...block({ className })}>
```

Comment:
I think we don't need `<div {...rest} {...block({ className })}> at all. In this case `className` is also redundant. @eszthoff what do you think about removing `div` wrapper and `className` prop? It will be breaking changes then. Pushed changes.

Reply:
Why is `className` redundant? We need that and we use that

Reply:
We don't use `className` approach for other Selects components (Autosuggest, ComboboxMulti), so I thought it is not used in our applications. I added  `className` to `SelectBase` and then `Select`.